### PR TITLE
List component

### LINKED ISSUE #35: Adapt list component
# User Story
As a **programmer**,
I want to **adjust the current list components**,
so I can **adapt to my design**.

# Description
image

#Material

# Done

- [ ] Add icons
- [ ] Add creation date
- [ ] Add item count (?) 
- [ ] Add styling and padding (!)

### PR DESCRIPTION
**Pull request by LebkuchenCJ**
Fix #35 

## Deployment: (click sign in to see result)
https://mega-app2020-list-compo-18p2fp.herokuapp.com/home

## Comments: 
 - Changed the design of the displayed shopping lists
 - Added onCheck mark
 - overflow not working (WHY?)

As a **programmer**,
I want to **adjust the current list components**,
so I can **adapt to my design**.

image

### CODE REVIEW

**Comment by aruppert** [comment]
In your screenshot everything looks fine. In Mozilla on Mac @ iPhone6/7/8 I get this:
Screenshot 2020-09-22 at 09 41 29
Adjusting the width of the divs in dev tools fixed the cut off shadow ;)

File: client/src/components/List.js [hunk 1]
```diff
@@ -1,11 +1,8 @@
...
-import PropTypes from "prop-types";
+import styled from "@emotion/styled";
 
-function List({ children }) {
-  return <div>{children}</div>;
-}
+const List = styled.div`
+  width: 90%;
+  overflow: scroll;
+`;
```
Comment (aruppert) [inline client/src/components/List.js:6]: You're missing a height. If you set a height value the scrollbar will appear ;)

**Review by aruppert — COMMENTED** [review]
Rest lgtm :)

File: client/src/components/List.js (hunk 1 above)
Comment (LebkuchenCJ) [reply client/src/components/List.js:6]: This has no effect on my component :/

**Pull request merged**